ui/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "feathr-ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@azure/msal-browser": "^2.15.0",
+    "@azure/msal-react": "^1.0.1",
+    "antd": "^4.3.0",
+    "axios": "^0.27.2",
+    "dagre": "^0.8.5",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-flow-renderer": "^10.2.0",
+    "react-query": "^3.2.0",
+    "react-resizable": "^3.0.4",
+    "react-router": "^5.0.1",
+    "react-router-dom": "^5.1.2"
```

Comment:
license of these dependencies should be in the description.

Reply:
Add a section in pr description to cover license. Basically, all down streaming dependencies are MIT licensed.